README.md

```diff
@@ -74,9 +74,26 @@ Design and goals of the project:
 This job templates and packages a given `chart` from the helm directory and
 pushes it to `app_catalog` for tagged builds and `app_catalog_test` otherwise.
 
-**NOTE**: It requires `CATALOGBOT_SSH_KEY_PRIVATE_BASE64` environment variable
-to be set in the build. This must be base64 encoded private SSH key of
-[CatalogBot github user](https://github.com/catalogbot).
+**NOTE**: The job requires `CATALOGBOT_SSH_KEY_PRIVATE_BASE64` environment
+variable to be set in the build. This must be base64 encoded private SSH key of
+[CatalogBot Github user][catalogbot-user].
+
+**NOTE**: App catalog repositories configured in the job parameters must be
+added to the [Catalog Editors][catalog-editors-team] GitHub team. See the
+paragraph below for explanation.
+
+This job assumes that the App Catalog is defined in a GitHub repository inside
+giantswarm organization. E.g. when `app_catalog` parameter is set to
+`"control-plane-test-catalog"` the job will try to use catalog
+https://github.com/giantswarm/control-plane-test-catalog. All interactions with
+the App Catalog GitHub repository are done with [CatalogBot github
+user] credentials.
+
+Detailed instructions on how to set up App Catalog can be found here:
+https://github.com/giantswarm/giantswarm/blob/master/processes/appcatalog.md#setting-up-a-new-app-catalog.
```

Comment:
@sslavic I linked here to the guide provided by you. I still want this to be merged. There are peaces of information that are implementation details not present in the other guide. And the other guide is too broad to included here.